includes/data-stores/class-wc-admin-reports-products-data-store.php

```diff
@@ -157,6 +160,10 @@ protected function include_extended_info( &$products_data, $query_args ) {
 						$extended_info[ $extended_attribute ] = $value;
 					}
 				}
+				// If there is no set low_stock_amount, use the one in user settings.
+				if ( '' === $extended_info['low_stock_amount'] ) {
+					$extended_info['low_stock_amount'] = absint( max( get_option( 'woocommerce_notify_low_stock_amount' ), 1 ) );
```

Comment:
Is the fallback value of `1` what is used in Woo Core too?

Reply:
This is the same logic that will be introduced in https://github.com/woocommerce/wc-admin/pull/974. 

When the first of these PRs lands, it will be a good opportunity to consolidate into a util function.